frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx

```diff
@@ -396,11 +396,12 @@ describe('MotiInventoryContainer functionality', () => {
       isPidAvailable.mockResolvedValue({ available: true });
       jest.clearAllMocks();
     });
+
     it('the cancel and submit buttons are disabled by default', async () => {
       const { findByText } = renderContainer({});
 
-      const cancelButton = await findByText('Cancel');
-      const submitButton = await findByText('Save');
+      const cancelButton = (await findByText('Cancel')).closest('button') as HTMLButtonElement;
+      const submitButton = (await findByText('Save')).closest('button') as HTMLButtonElement;
```

Comment:
This was finding a div within the button. Need to find the closest parent which is a **button**